cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -634,3 +634,42 @@ def _revert():
 
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.tier(2)
+def test_service_ansible_verbosity(
+    request,
+    ansible_catalog_item,
+    ansible_service_catalog,
+    ansible_service_request,
+    ansible_service,
+):
+    """Check if the different Verbosity levels can be applied to service and
+    monitor the std out
+
+    Bugzilla:
+        1460788
```

Comment:
please add automate marker for this BZ. as its RFE and you are automating this.

Reply:
@digitronik I have updated PR with your changes.